ansible/roles/schulcloud-server-core/templates/admin-api-server-deployment.yml.j2

```
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: admin-api-deployment
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: admin-api-serv
```

Name shoud be **api-admin**
```suggestion
    app: api-admin
```

---

done